src/filemenu/filemenu_pal.c

```diff
@@ -75,9 +75,7 @@ MenuWindowBP D_filemenu_8024F1D8[] = {
         .fpUpdate = { .func=&filemenu_update_show_title },
         .extraFlags = 0,
         .style = { .customStyle = &filemenu_windowStyles[3] }
-    },
-
-    {
+    }, {
```

Comment:
I think the style we've gone for before is this:
```c
{
    // ...
},
{
    // ...
},
```
but this is fine!

Reply:
Oh, I copied the style from the single `MenuWindowBP[]` that does it differently. Fixed.